x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/slm/SlmHealthIndicatorService.java

```diff
@@ -38,6 +39,10 @@ public class SlmHealthIndicatorService implements HealthIndicatorService {
     public static final String NAME = "slm";
 
     public static final String HELP_URL = "https://ela.st/fix-slm";
+    public static final UserAction SLM_NOT_RUNNING = new UserAction(
+        new UserAction.Definition("slm-not-running", "Snapshot lifecycle management is not running", HELP_URL),
```

Comment:
Similar to the ILM PR, lets say that they should enable SLM here instead of just repeating the indicator status.